utkdigitalinitiatives/iiif_assemble
# Address Digital-1299.

## What Does This Do

If a subject has a value of "narrator class", it's treated as its own thing in a manifest and has its own field.  Right now, this simply carries the same value from the MODS as its label, but this can change if needed.

## How Can I Test

The best way is to follow the instructions laid our in the [developer's setup](https://github.com/utkdigitalinitiatives/iiif_assemble#ut-libraries-developer-setup) instructions in the README. 

Minimally, read the logic in the change and think about how the xpath should be processed and if this is "correct."
